src/tree_cmd.cpp

```diff
@@ -249,6 +288,7 @@ void PlaceTreesRandomly()
 
 	i = Map::ScaleBySize(DEFAULT_TREE_STEPS);
 	if (_game_mode == GM_EDITOR) i /= EDITOR_TREE_DIV;
+	if (_settings_game.game_creation.tree_placer == TP_IMPROVED) i /= IMPROVED_TREES_DIV;
```

Comment:
Needs a comment to explain that we plant a limited number of "alone trees" outside existing forests.

Reply:
"alone trees" seems a rather odd phrase. if you have any recommendations for a better one i would be interested to hear them.